cadquery/occ_impl/shapes.py

```diff
@@ -959,7 +976,14 @@ def endPoint(self) -> Vector:
 
         return Vector(curve.Value(umax))
 
-    def tangentAt(self, locationParam: float = 0.5) -> Vector:
+    def paramAt(self: Mixin1DProtocol, d: float) -> float:
+
```

Comment:
Can I suggest a doc string for this one? It was difficult to figure out what this does.
```
Compute length along the edge at the specified location.
:param d: location to use in [0, 1]
:return: length
```

Reply:
I added a docstring, but note that it actually does something subtly different.

Reply:
Ah, now I think I understand. I wasn't clear on the definition of "parameter". For anyone else reading this, here is the definition of "parameter" for a circle in OCCT: https://old.opencascade.com/doc/occt-7.4.0/refman/html/class_geom___circle.html#details